lib/encrypt_env_variables.js

```diff
@@ -0,0 +1,37 @@
+import request from 'superagent-promise';
+import openpgp from 'openpgp';
+import Debug from 'debug';
+
+let debug = Debug('docker-worker-ci:encryptEnvVariable');
+
+export async function encryptEnvVariables(task, envs, publicKeyUrl) {
+  let response = await request.get(publicKeyUrl).buffer().end();
```

Comment:
Assuming this runs on heroku it's restarted every 24 hours... So if this was more than just docker-worker PRs, which is very very low volume, we should probably fetch the keyfile on start up, or bake it into the server..